integration-tests/src/tests/client/sharding_upgrade.rs

```diff
@@ -67,7 +67,8 @@ impl TestShardUpgradeEnv {
         let validators: Vec<AccountId> =
             (0..num_validators).map(|i| format!("test{}", i).parse().unwrap()).collect();
         let initial_accounts =
-            [validators, gen_unique_accounts(&mut rng, num_init_accounts)].concat();
+            [validators, gen_unique_accounts(&mut rng, num_init_accounts, num_init_accounts)]
```

Comment:
Not sure if I understand this change.  If my understanding is correct, before we would've generated accounts in the range [1, num_init_accounts+1), how we are going to generate accounts in the range [num_init_accounts, num_init_accounts].  Is that indeed what we want?

Reply:
Ah, I just read your comment and this change makes sense now.

Reply:
From my understanding of the tests in this file and the name of this argument, this is what we actually want.